gcp/modules/gcp-stackdriver-alerting/main.tf

```diff
@@ -28,8 +30,14 @@ resource "null_resource" "apply_stackdriver_alerting" {
     command = <<EOF
       export PROJECT_ID=${var.project_id}
       export GOOGLE_CLOUD_KEYFILE=${var.serviceaccount_key}
-      # export DEBUG=1
-      ruby ${path.module}/client.rb
+      # export STACKDRIVER_DEBUG=1
+      ruby -e '
+        require "${path.module}/client.rb"
+        resources = read_resources
```

Comment:
I like this refactor, but I don't like this logic being in the Terraform file. How about?

```
require client.rb
apply_resources
```

and

```
require client.rb
destroy_resources
```

This keeps the ruby logic in the ruby code while clarifying and simplifying everything by removing the `DESTROY=1` trick.

Reply:
There is no `DESTROY=1` trick anymore, but keeping all logic in `client.rb` file sounds good anyways.